index.html

```diff
@@ -0,0 +1 @@
+Hello there, awesome person!
```

Comment:
<html>
<body>
<p>
Good morning,
and thanks for the correction, cheers!
</p>
</body>
</html>